### PR TITLE
SAK-49109 T&Q: Images attached to parts display twice

### PR DESCRIPTION
**Pull request by hornersa**
A demo of this proposed fix is attached to the [corresponding jira](https://sakaiproject.atlassian.net/browse/SAK-49109).

### CODE REVIEW

File: library/src/skins/default/src/sass/modules/tool/samigo/_samigo.scss [hunk 1]
```diff
@@ -466,6 +466,10 @@
 	sakai-user-photo {
 		margin-right: 7px;
 	}
+
+	.portletBody {
+	    margin-bottom: 3rem !important;
+	}
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/samigo/_samigo.scss:472]: I believe we should utilize the class defined in `modules/tool/_base.scss`. The class already exists; we simply need to incorporate the new margin.

**Review by kunaljaykam — COMMENTED** [review]
Everything else looks good to me. Thanks, @hornersa

File: library/src/skins/default/src/sass/modules/tool/samigo/_samigo.scss (hunk 1 above)
Comment (hornersa) [reply library/src/skins/default/src/sass/modules/tool/samigo/_samigo.scss:472]: @kunaljaykam - Thanks. My initial intent was to isolate this change solely to T&Q. Upon looking at views in other tools (e.g., Discussions) it looks like the margin-bottom would be welcome there as well. I'll force push a revision imminently.

**Review by kunaljaykam — APPROVED** [review]
GTG!

**Pull request merged**